src/renderer/vt/VtSequences.cpp

```diff
@@ -456,3 +456,41 @@ using namespace Microsoft::Console::Render;
 {
     return _Write("\x1b[?9001h");
 }
+
+// Method Description:
+// - Formats and writes a sequence to add a hyperlink to the terminal buffer
+// Arguments:
+// - The hyperlink URI
+// Return Value:
+// - S_OK if we succeeded, else an appropriate HRESULT for failing to allocate or write.
+[[nodiscard]] HRESULT VtEngine::_SetHyperlink(const std::wstring_view& uri, const std::wstring_view& customId) noexcept
+{
+    // Opening OSC8 sequence
+    // We use the BEL character to terminate because \x9c gets mangled in std::string

```

Comment:
I think we should probably be using the standard 7-bit string terminator (`ESC \`). `BEL` is non-standard and not all terminals will handle that.